modules/appeals_api/app/swagger/contestable_issues/v0/api_description.md

```diff
@@ -1,16 +1,18 @@
-The Contestable Issues API lets internal VA teams retrieve a list of a claimant’s contestable issues and any chains of preceding issues. Contestable issues are issues from claims about which VA has made a decision that may be eligible for appeal. Not all contestable issues are guaranteed to be eligible for appeal; for example, claimants may have another appeal in progress for an issue.
+The Contestable Issues API lets you retrieve a list of a claimant’s contestable issues and any chains of preceding issues. Contestable issues are issues from claims about which VA has made a decision that may be eligible for appeal. Not all contestable issues are guaranteed to be eligible for appeal; for example, claimants may have another appeal in progress for an issue.
 
 To check the status of all decision reviews and appeals for a specified individual, use the [Appeals Status API](https://developer.va.gov/explore/appeals/docs/appeals?version=current).
 
 To file an appeal or decision review, use one of these APIs: 
-* `Higher-Level Reviews API`
-* `Notice of Disagreements API`
-* `Supplemental Claims API`
+* [Higher-Level Reviews API](https://developer.va.gov/explore/appeals/docs/higher_level_reviews)
+* [Notice of Disagreements API](https://developer.va.gov/explore/appeals/docs/notice_of_disagreements)
+* [Supplemental Claims API](https://developer.va.gov/explore/appeals/docs/supplemental_claims)
```

Comment:
These URLs are not active yet - we will need to make sure these are correct before these docs are made live